Animations/Fade/Down.js

```diff
@@ -26,5 +28,5 @@ var FadeDown = {
     'transform': 'translateY(-20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeDown', FadeDown);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeDown', FadeDown);
```

Comment:
There is a discrepancy throughout the prebuilt animations.  In them the `className` is being passed in as the first property rather than the second property.  It looks as though the `makeAnimation` function is expecting to see the `className` as the second argument.  Either the order for the prebuilt animations needs to be updated or these need to be updated.  I think the issue is that Spencer expected to be able to pass in a `displayName` as the first argument.  If that is no longer the case than just remove this `'FadeDown'` string input and do so for all the animations.